RxCocoa/iOS/Proxies/RxSearchBarDelegateProxy.swift

```diff
@@ -14,19 +14,33 @@ import UIKit
 import RxSwift
 #endif
 
-class RxSearchBarDelegateProxy : DelegateProxy
+public class RxSearchBarDelegateProxy : DelegateProxy
                                , UISearchBarDelegate
                                , DelegateProxyType {
     
-    class func currentDelegateFor(object: AnyObject) -> AnyObject? {
+    public class func currentDelegateFor(object: AnyObject) -> AnyObject? {
         let searchBar: UISearchBar = castOrFatalError(object)
         return searchBar.delegate
     }
     
-    class func setCurrentDelegate(delegate: AnyObject?, toObject object: AnyObject) {
+    public class func setCurrentDelegate(delegate: AnyObject?, toObject object: AnyObject) {
         let searchBar: UISearchBar = castOrFatalError(object)
         searchBar.delegate = castOptionalOrFatalError(delegate)
     }
+
+    // MARK: Delegate proxy methods
+    
+#if os(iOS)
```

Comment:
UISearchBar cannot be directly instantiated on tvOS. This makes it un-subclassable. You can't call super.init(). In theory it could be available to tvOS, but it's untestable in the current methods we use to test.